content/docs/user-guide/serving/index.md

```diff
@@ -23,32 +23,34 @@ INFO:     Application startup complete.
 INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
 ```
 
-Servers automatically create endpoints from model methods with payload schemas
-corresponding to serialized dataset types.
+The server is now running and listening for requests on the URL shown
+above.Endpoints are created automatically from model methods (using the
+`sample_data` [provided earlier](#saving-your-model)). You can open the
```

Comment:
Just some edits for consistency with the language used in the GS.

- [ ] But TBH [this page](https://mlem.ai/doc/user-guide/serving) as well as the [Building](https://mlem.ai/doc/user-guide/building) and [Deploying](https://mlem.ai/doc/user-guide/deploying) index pages are not in guide format. If I remember correctly they are copies from an old versions of the Get Started... Should rework 😄